src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -180,7 +181,7 @@ private void ComputeParameters()
             var diagnostics = BindingDiagnosticBag.GetInstance(_declarationDiagnostics);
 
             var parameters = ParameterHelpers.MakeParameters(
-                _binder,
+                ParameterBinder,
```

Comment:
>ParameterBinder [](http://example.com/codeflow?start=16&length=15)

Consider renaming corresponding parameter of ```ParameterHelpers.MakeParameters``` to clearly reflect what kind of binder it expects.